cromwell.examples.conf

```diff
@@ -115,6 +115,18 @@ call-caching {
   # to fail for external reasons which should not invalidate the cache (e.g. auth differences between users):
   # (default: true)
   #invalidate-bad-cache-results = true
+
+  # blacklist-cache {
+  #   # The call caching blacklist cache is off by default. This cache is used to blacklist cache hit paths based on the
+  #   # prefixes of cache hit paths that Cromwell has previously failed to copy for permissions reasons.
+  #   enabled: true
+  #   # Guava cache concurrency.
+  #   concurrency: 10000
+  #   # How long entries in the cache should live from the time of their last access.
+  #   ttl: 20 minutes
```

Comment:
Yes. Someone could change permissions on their bucket and having a `ttl` would allow Cromwell to pick up on that. Or one could configure some enormous value and effectively rule that out. This particular value was chosen to be fairly small mostly to start this discussion. 🙂